molecule/pki/host_vars/tls-ats-instance.yaml

```diff
@@ -0,0 +1,10 @@
+# placeholder
+ansible_user: ansible
+known_urls:
+  - http://localhost:8080/share
+  - https://someotherapp/
+acs:
+  artifact_name: alfresco-content-services-community-distribution
+  edition: Community
+  repository: "{{ nexus_repository.development_releases }}"
+  version: 7.4.0-A27
```

Comment:
we should avoid having versions in molecule tests or we will continue to forget about them - why is necessary here?

Reply:
leftovers. I had this file previously during development as I thought I would make this scenario a community one. I gave up with this idea. Will remove.